### PR TITLE
adding cisco dna center links

### PR DESCRIPTION
**Pull request by robertcsapo**
## Vendor Submission
Cisco DNA Center, Cisco IOS-XE, Cisco DevNet Event

## Explanation
Overall good documentation when working with Cisco DNA Center, Cisco IOS-XE.

_PS. Adding 2 Go struct converters for YAML and JSON._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,6 +289,7 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
  - [pyeapi](https://github.com/arista-eosplus/pyeapi) - Python library for Arista EOS.
  - [pyfg](https://github.com/spotify/pyfg) - Python library for Fortinet.
  - [pyiosxr](https://github.com/fooelisa/pyiosxr) - Python library for Cisco IOSXR.
+ - [dnacentersdk](https://github.com/cisco-en-programmability/dnacentersdk) - Python library for Cisco DNA Center Platform API
```
Comment (itdependsnetworks) [inline README.md:292]: Can you put in alphabetical order?

File: README.md [hunk 2]
```diff
@@ -162,15 +163,19 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 
  - [Awesome Go](https://github.com/avelino/awesome-go) - A curated list of awesome Go frameworks, libraries, software and resources.
  - [Getting Started and Basics](https://tour.golang.org/welcome) - Introduction to programming with Go.
+ - [YAML-to-Go](https://zhwt.github.io/yaml-to-go/) - YAML to Go structs
```
Comment (itdependsnetworks) [inline README.md:166]: Can you create complete sentences and periods to all updates

File: README.md (hunk 1 above)
Comment (robertcsapo) [reply README.md]: sure

File: README.md (hunk 2 above)
Comment (robertcsapo) [reply README.md]: does this look better?

**Pull request merged**